recipe/meta.yaml

```diff
@@ -1,27 +1,27 @@
 {% set version = "1.0.9" %}
-{% set commit = "cb3b00fd5e23eb6763012582d2bc8858809267d7" %}
+{% set commit = "b913aac7a25495caaa253ab4570f50a398e5639c" %}
 
 package:
   name: pyspharm
   version: {{ version }}
 
 source:
   url: https://github.com/jswhit/pyspharm/archive/{{ commit }}.tar.gz
-  sha256: dec9f360aef8b45ca59b2401cba8e9a9bcde7d314a90d51bb76fbd27a4e9a16f
+  sha256: 726e8884aa598c4e3ee959cd3ca64d4748818ff277b41d48523cd6d70882ceb9
 
 build:
-  number: 2
+  number: 3
   skip: True  # [win and py3k or win64]
 
 requirements:
   build:
     - python
-    - numpy 1.8.*
+    - numpy 1.11.*
     - mingwpy  # [win]
```

Comment:
@isuruf do you know if it is possible to use `flang` here instead of the outdated `mingwpy`?